src/Components/Login/Login.tsx

```diff
@@ -19,56 +13,61 @@ const ALL_USERS = gql `
   }
 `
 
-const ALL_BOOKS = gql `
-  query AllBooks {
-    books {
-      id
-      imageUrl
-    }
-  }
-`
-
 const Login = () => {
   const [email, setEmail] = useState('')
   const [username, setUsername] = useState('')
   const [userLocation, setUserLocation] = useState('')
   const [activeAccount, setAccount] = useState(true)
-  const [allUsers, setUsers] = useState([{userName: "Josh", emailAddress: "test", location: "Denver"}])
+  const [loginError, setLoginError] = useState(false)
+  const [errorMessage, setErrorMessage] = useState('')
+  const [allUsers, setUsers] = useState([{emailAddress: "joshua@gmail.com", userName: "Joshua", location: "Colorado"}])
 
   let navigate = useNavigate()
 
-  // const { data, loading, error } = useQuery(ALL_USERS)
-  const { data, loading, error } = useQuery(ALL_BOOKS)
-
-  console.log("Endpoint", error)
+  const { data, loading, error } = useQuery(ALL_USERS)
   
-
   const verifyLogin = () => {
      let noUser = allUsers.map(user => {
       if(user.emailAddress !== email){
-        setAccount(false)
+        setLoginError(true)
+        setErrorMessage("Could not find that email please try again")
+        setEmail('')
       } else {
         setAccount(true)
         navigate("/home")      
       }
-      
-     })
-     return noUser
+    }) 
+      return noUser
   }
 
   const createUser = () => {
-    let newUser = {userName: username, emailAddress: email, location: userLocation}
-
-    console.log(newUser);
-  
+    if(username === '') {
+      setLoginError(true)
+      setErrorMessage("Please fill in a username")
+    }
+    if(email === '') {
+      setLoginError(true)
+      setErrorMessage("Please fill in an email")
+    }
+    if(userLocation === '') {
+      setLoginError(true)
+      setErrorMessage("Please fill in a location")
+      console.log("Email: ", email, "Username: ", username);
+      
+    } else {
+      let newUser = {userName: username, emailAddress: email, location: userLocation}
```

Comment:
You initialize the `newUser` object here and then don't do anything with it. Is there a future plan?

Reply:
There is. I will be using a mutation to create a new user and add it to the users query.